### ISSUE TITLE
[DOC] Feedback on Search topics

### ISSUE THREAD

**Issue by hdhalter** [opensearch-project/documentation-website#4326]
**What do you want to do?**
 
x Request a change to existing documentation

**Tell us about your request.** Provide a summary of the request and all versions that are affected.
 Feedback from the community on Search topics:

- [ ] 20230501 | /docs/latest/search-plugins/knn/knn-index/ |The documentation doesn't provide a way how we can create an index with method as IVF and engine as Faiss. There is no sample payload. It is unusable
- [x] 20230606 | /docs/latest/search-plugins/search-pipelines/index/ | Missing the flags needed to enable the experimental feature for testing.
- [ ] 20230504 |/docs/latest/search-plugins/searching-data/paginate/ |none of the methods are not working, when I put from as 8000 and size as 10000 the throwing error , like they adding from and size together and saying it is more than 10000 .
- [ ] 20230525 |/docs/latest/search-plugins/searching-data/sort/ |Creating a sort filter on the Discovery page does not provide correct results.  Also, the Time column
- [ ] 20230410 |/docs/latest/search-plugins/sql/ppl/functions/ |How to search for a text
- [ ] 20230522 | /docs/latest/search-plugins/sql/sql/basic/ COUNT (distinct xxx) is not supported
- [ ] 20230522 |/docs/latest/search-plugins/sql/sql/complex/ |no support for distinct
- [ ] 20230428 |/docs/2.3/search-plugins/sql/index/ |0 examples, nothing useful other than the statement that SQL and PPL are supported
- [ ] 20230505 |/docs/2.3/search-plugins/sql/sql-ppl-api/ | Missing more information on _explain.
Like using 'WHERE field IN' parameters
- [ ] 20230302 |/docs/2.4/search-plugins/sql/sql/complex/ |subquery barely works. often doesn't work at all even for queries same to examples
- [ ] 20230425 |/docs/2.5/search-plugins/sql/full-text/ |I am looking for more detailed description about multi_match options. In this article you have only a list of options and a misleading statement in the bottom "Please, refer to multi_match query documentation for parameter description and supported values." which points to itself.
- [ ] 20230228 | /docs/2.5/neural-search-plugin/index/ | - Reverse order documentation to follow it
- Show me the pipeline name
- Try to add placeholder for

- [ ] 20230701 | /docs/latest/search-plugins/knn/approximate-knn/ | It's really not clear what the difference between the different knn engines are.
- [ ] 20230926 | /docs/latest/search-plugins/knn/performance-tuning/ | it's misleading doc. The disable refresh interval doesn't apply to permanents basis improvement. Need to add commands to all recommendation to help test them. Really disappointed.
- [ ] 20230928 | /docs/latest/search-plugins/knn/performance-tuning/ | no practical examples to show how to apply it.
- [ ] 20231002 | /docs/latest/search-plugins/neural-search/ | Query and index examples are disconnected, which makes the documentation hard to follow. One uses a field "passage_embedding" but the query uses "passage_vector". An example of a query that can be run on the example index would help connect the dots.
- [ ] 20230823 | /docs/latest/search-plugins/sql/ppl/syntax/ | Doesn't show how to search index wildcard. For example: accounts*
- [ ] 20230831 | /docs/latest/search-plugins/sql/index/ | How does one actually use this plugin?
- [ ] 20230906 | /docs/latest/search-plugins/sql/sql/complex/ | > You can only join two indexes.   What does that mean for nested documents? Can't you query them by using all their nested objects? Or this only applies to joining different collections of documents with no hierarchical relationship?
- [ ] 20230912 | /docs/latest/search-plugins/searching-data/index/ | I want to know how to filter data I am beginner and see nothing that will help with a basic search. I'm looking for a "hello world" version of searching for data
- [ ] 20230914 | /docs/latest/search-plugins/knn/knn-index/ | These tables are nice, but I was still left not knowing what a finished call looked like and what parts of the example calls needed to have parts filled in.
- [ ] 20230919 | /docs/2.7/search-plugins/sql/full-text/ | No example of exact match - or starts with or ends with type of queries
- [ ] 20230919 | /docs/2.9/search-plugins/neural-search/ | The 'passage_vector' field in the Example request should be replaced with the name of your vectorized field. You can check the details of your search pipeline to see it.
- [ ] 20230919 | /docs/2.7/search-plugins/sql/full-text/ | No example of exact match - or starts with or ends with type of queries
- [ ] 20230919 | /docs/2.9/search-plugins/neural-search/ | The 'passage_vector' field in the Example request should be replaced with the name of your vectorize d field. You can check the details of your search pipeline to see it.
- [ ] 20230921 | /docs/latest/search-plugins/search-pipelines/filter-query-processor/ | "error": "no handler found for uri [/_search/pipeline/my_filter] and method [PUT]"
- [ ] 20231106 | /docs/latest/search-plugins/searching-data/autocomplete/ | How does one simply get started with the DSL? Do I literally have to write JSON in the search bar?
- [ ] 20231107 | /docs/latest/search-plugins/knn/knn-index | You don't tell me that dimension has a limit of 1024? Do you guys know how many hours I wasted on this?
- [ ] 20231112 | /docs/latest/search-plugins/sql/functions/ | It has no examples of using the functions

**What other resources are available?** Provide links to related issues, POCs, steps for testing, etc.
https://opensearch.org/docs/latest/search-plugins/index/

**Comment by hdhalter** [comment]
@kolchfa-aws , does this PR https://github.com/opensearch-project/documentation-website/pull/4476 address 20230501 above? Thanks.

**Comment by kolchfa-aws** [comment]
@hdhalter No, it doesn't

**Comment by kolchfa-aws** [comment]
@hdhalter This one is Dashboards-related and should be moved to the Dashboards feedback

- [ ] 20230525 |/docs/latest/search-plugins/searching-data/sort/ |Creating a sort filter on the Discovery page does not provide correct results. Also, the Time column

**Comment by hdhalter** [comment]
- [ ] 20231219 | /docs/2.11/search-plugins/sql/sql/partiql/ | I tried this with v2.11.1 but the query is retruning error: QueryShardException[failed to create query: [nested] nested object under path [projects] is not of nested type]
- [ ] 20231204 | /docs/latest/search-plugins/neural-search-tutorial/ | Add python code examples on how to use these.
- [ ] 20231222 | /docs/latest/search-plugins/searching-data/paginate/ | Scroll operation not clearly understandable, like "/_search?scroll=10m"in this meaning of sroll arg is not explained,
- [ ] 20231127 | /docs/latest/search-plugins/sql/ppl/syntax/ | with 3 basic example? is this a seroius question?

**Comment by natebower** [comment]
Closing as stale
